tools/theme/src/Theme.ts

```diff
@@ -522,11 +485,6 @@ function checkForIssues(
             }
         };
 
-        if (hasThemeAttribute) {
-            issues.push(
-                `DEPRECATION NOTICE: the "theme" attribute has been deprecated in favor of "system". For more information, see: https://opensource.adobe.com/spectrum-web-components/tools/theme/`
-            );
-        }
         if (['lightest', 'darkest'].includes(color || '')) {
```

Comment:
This can be remove?

Reply:
It is still supported but is deprecated, so I think we can keep the warning.